### PR TITLE
Branch out `releases` on `rc0`

### PR DESCRIPTION
**Pull request by mkolesnik**
Since we're creating stable branches on `rc0` we should create the
branch on `releases` as well, so that all stable releases happen on the
branch.

This also makes sure the releases on `releases` happen on the stable
branch.

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

**Comment by mkolesnik** [comment]
This should be backported to `release-0.12` and any other stable branch we intend to support

**Comment by nyechiel** [comment]
This breaks some of the releases testing. We can live without it for 0.12.2, but should merge before we tag 0.13.0-rc0.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-BASE_BRANCH ?= devel
+BASE_BRANCH = devel
```
Comment (skitt) [inline Makefile:1]: Why are these changes needed? (This one and hard-coding `devel` in line 72.)

**Review by skitt — CHANGES_REQUESTED** [review]

**Comment by mkolesnik** [comment]
Seems that the branching out must happen on theCI, so closing this in favor of (much simpler) #391

**Pull request closed without merging**